configs/crowbar.conf

```diff
@@ -9,8 +9,7 @@
     ErrorDocument 422 /422.html
     ErrorDocument 500 /500.html
 
-    ProxyPass / http://127.0.0.1:3000/ timeout=600 Keepalive=On
+    ProxyPass / http://127.0.0.1:3000/ timeout=1500 Keepalive=On
```

Comment:
I understand. thats why I recommended using the same string ("timeout=1500") for both locations so that simple search&replace will find both occurrences